test/abbrtypes.ml

```diff
@@ -3,16 +3,12 @@
    those values at all. Therefore, we copy the record type that use these and
    skip the problematic fields. *)
 
-(* These are copied directly from slacko.ml so we can use them here. *)
-type timestamp = float
-let timestamp_to_yojson ts = `Int (int_of_float ts)
-let timestamp_of_yojson = function
-  | `Int x -> Result.Ok (float_of_int x)
-  | `Intlit x -> Result.Ok (float_of_string x)
-  | `String x -> Result.Ok (float_of_string x)
-  | _ -> Result.Error "Couldn't parse timestamp type"
+open Slacko__Internal
+
+type timestamp = Ptime.t
+
 (* But this one is new. *)
-let pp_timestamp fmt ts = Format.pp_print_float fmt ts
+let pp_timestamp = Ptime.pp_human ~frac_s:6 ()
```

Comment:
It might be nice to make this into a `Timestamp` module, then the prettyprinter can just be `pp` and the code operating on `Timestamp.t` is somewhat contained. But no strong opinions on this one.